sdk/communication/azure-communication-identity/azure/communication/identity/_shared/utils.py

```diff
@@ -84,18 +87,20 @@ def create_access_token(token):
         raise ValueError(token_parse_err_msg)
 
     try:
-        padded_base64_payload = base64.b64decode(parts[1] + "==").decode('ascii')
+        padded_base64_payload = base64.b64decode(
+            parts[1] + "==").decode('ascii')
```

Comment:
Is this formatting idiomatic Python style or is it a mistake?

Reply:
this is what happens if you hit "format document"...so I suppose yes.